### PR TITLE
Query operations as an account connections

### PR DESCRIPTION
**Pull request by charlie-wasp**
This change is [Reviewable](https://reviewable.io/reviews/astroband/astrograph/129)

### CODE REVIEW

File: src/schema/accounts.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    inflationDest: AccountID
+    homeDomain: String
+    thresholds: AccountThresholds!
+    flags: AccountFlags!
+    ledger: Ledger!
+    signers: [Signer]
+    data: [DataEntry]
+    trustLines: [TrustLine]
+    signerFor(first: Int!): [Account!]
+    operationsConnection(first: Int, after: String, order: Order): AccountOperationsConnection
```
Comment (nebolsin) [inline src/schema/accounts.ts]:
```suggestion
    operations(after: String, before: String, first: Int, last: Int): OperationConnection!
```

File: src/schema/accounts.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  }
+
+  type AccountOperationsConnection {
```
Comment (nebolsin) [inline src/schema/accounts.ts]:
```suggestion
  type OperationConnection {
```

File: src/schema/accounts.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    pageInfo: PageInfo!
+    edges: [AccountOperationsEdge]
```
Comment (nebolsin) [inline src/schema/accounts.ts]:
```suggestion
    edges: [OperationEdge]
    nodes: [Operation]
```

File: src/schema/accounts.ts [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  }
+
+  type AccountOperationsEdge {
```
Comment (nebolsin) [inline src/schema/accounts.ts]:
```suggestion
  type OperationEdge {
```

File: src/schema/accounts.ts [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    cursor: String!
+    node: IOperation
```
Comment (nebolsin) [inline src/schema/accounts.ts]:
```suggestion
    node: Operation
```

File: src/schema/type_defs.ts [hunk 6]
```diff
@@ -2,12 +2,20 @@ import { gql } from "apollo-server";
...
   scalar MemoValue
   scalar DateTime
 
+  enum Order {
+    desc
+    asc
+  }
+
+  type PageInfo {
+    endCursor: String!
```
Comment (nebolsin) [inline src/schema/type_defs.ts]:
```suggestion
    endCursor: String
    hasNextPage: Boolean!
    hasPreviousPage: Boolean!
    startCursor: String
```

File: src/schema/accounts.ts (hunk 1 above)
Comment (charlie-wasp) [reply src/schema/accounts.ts]: Would it be better to keep `operationsConnection` name? In fact, it doesn't lead to the operations themselves, there is some proxy entity with `edges` and `pageInfo`. And I thought such naming is conventional 🤔

**Pull request merged**